src/draw/lv_draw_mask.c

```diff
@@ -1218,6 +1250,79 @@ LV_ATTRIBUTE_FAST_MEM static lv_draw_mask_res_t lv_draw_mask_map(lv_opa_t * mask
     return LV_DRAW_MASK_RES_CHANGED;
 }
 
+LV_ATTRIBUTE_FAST_MEM static lv_draw_mask_res_t lv_draw_mask_polygon(lv_opa_t * mask_buf, lv_coord_t abs_x,
```

Comment:
Does it support concave polygons like this?
![image](https://user-images.githubusercontent.com/7599318/147927493-dff6e233-bbd4-460c-b7d3-6dfcc3fb6cdb.png)

Reply:
It supports convex polygon only. Should LVGL support concave polygons?

Reply:
It should, but doesn't support at this moment :laughing: 

Supporting convex only is fine, as it's in pair with current features.

Reply:
I'll support this in later patches. So users can use mask directly anywhere (without point limit).